integration/benchmark/cmd/ci/main.go

```diff
@@ -163,57 +170,56 @@ func main() {
 	if err != nil {
 		log.Fatal().Err(err).Msgf("unable to init loader")
 	}
+
+	// run load
 	lg.Start()
```

Comment:
you may want to split this into 1) `Setup()` and 2) `Start()`.  That way you won't need the `leadDuration` delay.

PS. Given that there is already `Init()` you can move all the pre-benchmark code there and rename it to `Setup`.

Reply:
I have removed `leadDuration` entirely as it wasn't providing much context or added data. If we find that the first and last slices are not providing useful data it will be easier to discard them after calculation rather than try to split the code up.